environment.yml

```diff
@@ -45,7 +45,12 @@ dependencies:
   - autodocsumm>=0.2.2
   - ipython
   - nbsphinx
-  - sphinx>=6.1.3
+  # sphinx >=6 can not be installed because
+  # sphinx_rtd_theme are pinning docutils<0.19 which is the only
```

Comment:
Maybe we could use the theme suggested by @zklaus in another issue (can't find it right now)?

Reply:
I suggested that as a remedy for this problem here in #2004, which is a PR targeting this branch.

Reply:
gents, yes - but let's stick to the scope of this PR - #2004 (or, its previous incarnation) was merged here - through which good old Klaus fixed the RTD crappe that gave me plenty of headaches. Let's see about the new theme in a separate PR though, chances are that may be merged before this one anyway :grin:

Reply:
We can do that, though, as you noted, sphinx_rtd_theme breaks Python 3.11 compatibility.

Reply:
well, not for sphinx<6 - can we live with sphinx=5 for a while til we get our new theme up and spick and span?

Reply:
Hm. That breaks (at least) the API docs for `esmvalcore.cmor.check.CheckLevels`, though maybe we can live without them for a bit?

Reply:
yeah was thinking that too

Reply:
Done